test/unit/calculators/benefit_cap_calculator_configuration_test.rb

```diff
@@ -127,7 +127,7 @@ class BenefitCapCalculatorConfigurationTest < ActiveSupport::TestCase
           end
         end
         context "merge questions" do
-          should "get all benefit questions from multiple configuration sets" do
+          should "get all unique benefit questions from multiple configuration sets" do
```

Comment:
I agree. I decided against it partly due to time available and partly as I wanted to reuse as many commits as possible from PR #2542.